improver/calibration/simple_bias_correction.py

```diff
@@ -248,11 +251,59 @@ class ApplyBiasCorrection(BasePlugin):
     the specified bias values.
     """
 
-    def __init__(self):
+    def __init__(
+        self,
+        lower_bound: Optional[float] = None,
+        upper_bound: Optional[float] = None,
+        fill_masked_bias_values: Optional[bool] = False,
+    ):
         """
         Initialise class for applying simple bias correction.
+
+        Args:
+            lower_bound:
+                A lower bound below which all values will be remapped to
+                after the bias correction step.
+            upper_bound:
+                An upper bound above which all values will be remapped to
+                after the bias correction step.
+            fill_masked_bias_values:
+                Flag to specify whether masked areas in the bias data
+                should be filled to an appropriate fill value.
         """
         self.correction_method = apply_additive_correction
+        self.lower_bound = lower_bound
+        self.upper_bound = upper_bound
+        self.fill_masked_bias_values = fill_masked_bias_values
```

Comment:
Make these all 'private' (i.e. leading underscore), including the existing 'correction_method' to hide data within the class.  If we want to expose some data to users then we can do so using properties or similar (which I don't think we need to in this case).